tests/epyccel/test_lists.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+import inspect
+import pytest
+from modules import lists as lists_module
+from pyccel import epyccel
+
+list_funcs = [(f, getattr(lists_module,f)) for f in lists_module.__all__ if inspect.isfunction(getattr(lists_module,f))]
+
+marks = [f[1] for f in list_funcs]
+
+@pytest.mark.parametrize('test_func',marks)
+@pytest.mark.parametrize( 'language', (pytest.param("python", marks = pytest.mark.python),))
+def test_lists(test_func, language):
+    f1 = test_func
+    f2 = epyccel( f1 , language=language)
+
+    python_out = f1()
+    pyccel_out = f2()
+    print(pyccel_out)
+    print(python_out)
+
+    assert python_out == pyccel_out
```

Comment:
We already have a file for testing list functions. It is here:
https://github.com/pyccel/pyccel/blob/devel/tests/epyccel/test_epyccel_lists.py

Please can you move this function there so we have all list functions in one place